packages/benchmarks/src/models.ts

```diff
@@ -6,12 +6,7 @@ export type ModelDeveloper =
   | "Mistral"
   | "Amazon";
 
-export type ModelProvider =
-  | "radiant"
-  | "braintrust"
-  | "azure_openai"
-  | "aws_bedrock"
-  | "gcp_vertex_ai";
+export type ModelProvider = "braintrust" | "azure_openai" | "gcp_vertex_ai";
```

Comment:
Not against keeping `azure_openai` as an option - just confirming that we won't actually be using it for now?

Reply:
oh we can remove. adding to PR

```suggestion
export type ModelProvider = "braintrust" | "gcp_vertex_ai";
```